service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/get.symlink.out.of.sandbox.rpt

```diff
@@ -0,0 +1,6 @@
+connect http://localhost:8701/../../../../link
+connected
+
+write method "GET"
+
+read status "404" "Bad Request"
```

Comment:
Not sure how this is testing something different then continuous.symlinks.into.directory.from.outside.rpt